test/transform/BodySpacingTransform.test.js

```diff
@@ -0,0 +1,50 @@
+import { BodySpacingTransform } from '../../build/wikimedia-page-library-transform'
+import assert from 'assert'
+import domino from 'domino'
+
+describe('BodySpacingTransform', () => {
+  describe('.setMargins()', () => {
+    it('no values', () => {
+      const document = domino.createDocument('<p></p>')
+      BodySpacingTransform.setMargins(document, {})
+      assert.strictEqual(document.body.outerHTML, '<body><p></p></body>')
+    })
+
+    it('just left + right', () => {
+      const document = domino.createDocument('<p></p>')
+      BodySpacingTransform.setMargins(document, { right: '8px', left: '16px' })
+      const style = 'margin-right: 8px; margin-left: 16px;'
+      assert.strictEqual(document.body.outerHTML,`<body style="${style}"><p></p></body>`)
```

Comment:
This could fail in future if domino changes the whitespace it uses to separate inline styles components, or the component order. May consider separate asserts for expected `document.body.style.marginRight` and `document.body.style.marginLeft` values.

Reply:
Same for other asserts below...

Reply:
Done